content/tutorials/guides/how-to-collect-product-variations.md

```diff
@@ -0,0 +1,277 @@
+<a href="/tutorials/homepage.html" class="back-button">
+   <button>
+      <img src="/img/icons/icon--arrow-back.svg" style="margin-right: 10px;">
+      All guided tutorials
+   </button>
+</a>
+
+# How to collect product variations
+
+<table class="tag-container">
+    <tr>
+        <td>Use case:</td>
+        <td>
+            <div class="tag-not-selectable">
+                <div class="tag-color tag-color-light-blue"></div>
+                <div class="tag-label">App Workflow</div>
+            </div>
+        </td>
+    </tr>
+    <tr>
+        <td>PIM Features:</td>
+        <td class="td-features">
+            <div class="tag-not-selectable">
+                <div class="tag-color tag-color-purple"></div>
+                <div class="tag-label">Product Models</div>
+            </div>
+            <div class="tag-not-selectable">
+                <div class="tag-color tag-color-light-green"></div>
+                <div class="tag-label">Variant Products</div>
+            </div>
+        </td>
+    </tr>
+</table>
+
+<div class="endpoint-container">
+    <div class="endpoint-text">REST API endpoint(s):</div>
+    <a href="/api-reference.html#get_families" class="endpoint-link" target="_blank" rel="noopener noreferrer">family</a>
+    <a href="/api-reference.html#Familyvariants" class="endpoint-link" target="_blank" rel="noopener noreferrer">family variants</a>
+    <a href="/api-reference.html#Attribute" class="endpoint-link" target="_blank" rel="noopener noreferrer">attributes</a>
+</div>
+
+<div class="block-requirements">
+    <div class="block-requirements-headline">
+        If you're following our App workflow, make sure you previously read:
+    </div>
+    <div class="block-requirements-row">
+        <img src="../../img/illustrations/illus--Attributegroup.svg" width="110px">
+        <div class="block-requirements-steps">
+            <ul>
+                <li>Step 1. <a href="how-to-get-your-app-token.html" target="_blank" rel="noopener noreferrer">Get your App token tutorial</a></li>
+                <li>Step 2. <a href="how-to-retrieve-pim-structure.html" target="_blank" rel="noopener noreferrer">How to retrieve PIM structure</a></li>
+                <li>Step 3. <a href="how-to-get-families-and-attributes.html" target="_blank" rel="noopener noreferrer">How to get families and attributes</a></li>
+                <li>Step 4. <a href="how-to-get-product-information.html" target="_blank" rel="noopener noreferrer">How to get product information</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+
+## Context
+
+In the PIM we handle product models and product variations.
+
```

Comment:
Miss the tip "Before digging into the code you can find out more about these concepts in our [helpcenter](https://help.akeneo.com/pim/serenity/articles/what-about-products-variants.html#about-products-with-variants)." (See Notion)